content/testing-yaml/testing.md

```diff
@@ -7,6 +7,8 @@ aliases:
 ---
 
 Test scripts are added under `scripts` in the [overall architecture](../getting-started/yaml#template), before the build commands.
+You can display test results visually in the build overview if you use expanded form of the script in codemagic.yaml.
```

Comment:
This should be a new paragraph or even in a notebox.
... if you use an expanded form of the script in `codemagic.yaml`.

Reply:
started a new paragraph and fixed in e0d9c8e6222cf49670671fc10c0b345d6644767c